CHANGELOG.md

```diff
@@ -2,6 +2,9 @@
 
 ## [Unreleased]
 
+### Added
+- Introduced "remove" method for Lazy values ([#1238](https://github.com/near/near-sdk-rs/pull/1238))
```

Comment:
You don't need to add the CHANGELOG manually, just name the PR in such a way that release-plz can just use the commit/PR title as the CHANGELOG entry as is